src/index.js

```diff
@@ -0,0 +1,8 @@
+const provide       = require('./provide')
+const shouldProvide = require('./detect')
```

Comment:
Hm, having one called `detect` but using `shouldProvide` as the var name seems a bit much for two small files. I would think about just naming detect.js shouldProvide.js.